cryptoinvestor/main.py

```diff
@@ -176,7 +177,7 @@ def update_assets(self):
 
 def main(args=None):
     views.BaseView.app = App(config_file=args.config)
-    application.run()
+    application.run(debug = True)
```

Comment:
This should not be in production. Master is being built straight to heroku server. This would break the integration.